maskrcnn_benchmark/engine/trainer.py

```diff
@@ -70,15 +70,18 @@ def do_train(
         # reduce losses over all GPUs for logging purposes
         loss_dict_reduced = reduce_loss_dict(loss_dict)
         losses_reduced = sum(loss for loss in loss_dict_reduced.values())
-        meters.update(loss=losses_reduced, **loss_dict_reduced)
```

Comment:
Can we avoid this change? It could affect the reported runtime due to the async nature of CUDA operations.

Reply:
Just this change left, and then I believe this looks good to be merged!

Reply:
i just put it back

Reply:
Keeping the two updates separate will still cause the iteration count to be off by a factor of 2, right? Should we explicitly define and call meters.update_iter() or similar to only update the iteration count once per loop?